package.json

```diff
@@ -45,6 +45,7 @@
     ]
   },
   "dependencies": {
+    "@changesets/cli": "^2.26.1",
```

Comment:
Let's move this to devDeps. For some reason, I've noticed that `yarn install -D` tends to put this in the wrong place, and I have no idea why.